module/VuFind/src/VuFind/RecordDriver/Feature/MarcAdvancedTrait.php

```diff
@@ -890,12 +945,6 @@ public function getXML($format, $baseUrl = null, $linker = null)
 
             // Set up proper namespacing and extract just the <record> tag:
             $xml->record->addAttribute('xmlns', "http://www.loc.gov/MARC21/slim");
-            $xml->record->addAttribute(
```

Comment:
Why is this being removed? I don't doubt that there's a good reason for it, but it seems not to be directly related to the rest of this PR (unless I'm missing something).

Reply:
MarcXml serialization already added that. It wasn't quite right, though, since it's better to have it at the collection level. I changed it and reverted the change here. An additional problem was that this referred the xsi namespace without declaring it. I've now added definition for that, but it's a bit ugly due to a SimpleXML quirk.

Reply:
Thanks! And yes, I've run into that SimpleXML quirk too -- quite annoying.